doc/rbd/iscsi-requirements.rst

```diff
@@ -0,0 +1,53 @@
+==========================
+iSCSI Gateway Requirements
+==========================
+
+To implement the Ceph iSCSI gateway there are a few requirements. Red
+Hat recommends using two nodes for a highly available Ceph iSCSI gateway
+solution and up to four Ceph iSCSI gateway nodes.
+
+For hardware recommendations, see the `Hardware Recommendation page <http://docs.ceph.com/docs/master/start/hardware-recommendations/>`_
+for more details.
+
+.. WARNING::
+    On the iSCSI gateway nodes, the memory footprint of the RBD images
+    can grow to a large size. Plan memory requirements accordingly based
+    off the number RBD images mapped. Each RBD image roughly uses 90 MB
```

Comment:
The physical memory usage is more normal now. We will want around 8 MB per exported RBD image.

The virtual memory usage is still really high at 500 MB per image. I have not had a chance to look into that. I suspected it was just due to the ring buffer vmalloc and mmap use and maybe memory from that being accounted for incorrectly.